packages/kerosene-ui/src/hooks/usePopup.spec.tsx

```diff
@@ -22,16 +24,16 @@ Object.assign(window, {
 });
 
 describe("#usePopup", () => {
-  let clock: FakeTimers.InstalledClock;
-  beforeEach(() => {
-    clock = FakeTimers.install();
-  });
+  // let clock: FakeTimers.InstalledClock;
```

Comment:
can this commented out code be removed?

Reply:
Whoops, missed committing these changes. Added now